docs/site/src/demos/drawers/UndockedDrawer.js

```diff
@@ -0,0 +1,104 @@
+// @flow weak
+
+import React, { Component } from 'react';
+import { createStyleSheet } from 'jss-theme-reactor';
+import customPropTypes from 'material-ui/utils/customPropTypes';
+import Drawer from 'material-ui/Drawer';
+import Button from 'material-ui/Button';
+import {
+  List,
+  ListItem,
+  ListItemText,
+  ListItemIcon,
+} from 'material-ui/List';
+import Divider from 'material-ui/Divider';
+import InboxIcon from 'material-ui/svg-icons/inbox';
+import DraftsIcon from 'material-ui/svg-icons/drafts';
+import StarIcon from 'material-ui/svg-icons/star';
+import SendIcon from 'material-ui/svg-icons/send';
+import MailIcon from 'material-ui/svg-icons/mail';
+import DeleteIcon from 'material-ui/svg-icons/delete';
+import ReportIcon from 'material-ui/svg-icons/report';
+
+
+const styleSheet = createStyleSheet('UndockedDrawer', () => ({
+  list: {
+    width: '280px',
```

Comment:
noticed that width in specs, but then realized that width prop is missing in Drawer anyway, so havent bothered with this much now. Sorry, will change to master's value!